packages/foam-vscode/static/dataviz/graph.js

```diff
@@ -501,6 +501,9 @@ try {
       case 'didUpdateGraphData':
         graphData = augmentGraphInfo(message.payload);
         Actions.refreshWorkspaceData(graphData);
+        graph.zoom(graph.zoom() * 1.5);
+        graph.cooldownTicks(100);
+        graph.onEngineStop(() => graph.zoomToFit(500));
```

Comment:
does this mean that every time a file is saved the graph is put back into a zoom-to-fit state?